File: problems/c/cetvrta/cetvrta.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    fourth_x = []
    fourth_y = []
    for _ in range(3):
        x, y = [int(val) for val in input().split()]
        if x in fourth_x:
            fourth_x.remove(x)
        else:
            fourth_x.append(x)

        if y in fourth_y:
            fourth_y.remove(y)
        else:
            fourth_y.append(y)

    print(fourth_x[0], fourth_y[0])


if __name__ == '__main__':
    main()

File: problems/c/cetvrta/cetvrta.c
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <stdio.h>

int main(void) {
    int x, y;
    int x_points[3], y_points[3];
    int fourth_x, fourth_y;
    for (int row=0; row<3; row++) {
        if (scanf("%d %d", &x, &y) != 2) {
            printf("Failed to read a!");
        }
        x_points[row] = x;
        y_points[row] = y;
    }
    if (x_points[0] == x_points[1]) {
        fourth_x = x_points[2];
    }
    else if (x_points[0] == x_points[2]) {
        fourth_x = x_points[1];
    }
    else {
        fourth_x = x_points[0];
    }

    if (y_points[0] == y_points[1]) {
        fourth_y = y_points[2];
    }
    else if (y_points[0] == y_points[2]) {
        fourth_y = y_points[1];
    }
    else {
        fourth_y = y_points[0];
    }

    printf("%d %d\n", fourth_x, fourth_y);

    return 0;
}